### PR TITLE
Contact us page

### PR DESCRIPTION
**Pull request by alvikalvik**

### CODE REVIEW

File: src/components/ContactUs/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+              errors.email = 'Required';
+            } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)) {
+              errors.email = 'Invalid email address';
+            }
+
+            if (!values.message) {
+              errors.message = 'Required';
+            }
+            return errors;
+          }}
```
Comment (glebdronchenko) [inline src/components/ContactUs/ContactUs.jsx]: I think will be better separate this validate function to several functions and move it to another file.

File: src/components/ContactUs/ContactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          onSubmit={(values, { setSubmitting }) => {
+            setIsAfterSubmittiningMessageActive(true);
+            localStorage.setItem('feedback', JSON.stringify(values));
+            setSubmitting(false);
+
+            setTimeout(() => {
+              setIsAfterSubmittiningMessageActive(false);
+            }, AFTER_SUBMITTING_TIMEOUT);
+          }}
```
Comment (glebdronchenko) [inline src/components/ContactUs/ContactUs.jsx]: You can make this handle as separate function and pass link to this function.

File: src/components/ContactUs/ContactUs.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { useState } from 'react';
+import { AFTER_SUBMITTING_TIMEOUT } from '../../constants/constants';
+
+function ContactUs() {
+  const [isAfterSubmittiningMessageActive, setIsAfterSubmittiningMessageActive] = useState(false);
```
Comment (glebdronchenko) [inline src/components/ContactUs/ContactUs.jsx]: So long variable name. You can name this for example isFormSent or isFormResultActive.

Comment (anton-vlasik) [reply src/components/ContactUs/ContactUs.jsx]: 1) correct variant would be ...submitting...
2) shouldShowSubmitMessage

File: src/components/ContactUs/ContactUs.jsx (hunk 1 above)
Comment (anton-vlasik) [reply src/components/ContactUs/ContactUs.jsx]: Agree. It is a whole bunch of logic that is hard to percept.

File: src/components/ContactUs/ContactUs.jsx (hunk 3 above)
Comment (alvikalvik) [reply src/components/ContactUs/ContactUs.jsx]: indeed, renamed to shorter name. Thanks.

File: src/components/ContactUs/ContactUs.jsx (hunk 1 above)
Comment (alvikalvik) [reply src/components/ContactUs/ContactUs.jsx]: Thanks. I made a separate validateForm function and moved it to utils file.

File: src/components/ContactUs/ContactUs.jsx (hunk 2 above)
Comment (alvikalvik) [reply src/components/ContactUs/ContactUs.jsx]: Thanks. I made a separate handleFormSubmit function and moved it to utils file.

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**